.github/workflows/build-test-radosgw.yml

```diff
@@ -55,23 +54,55 @@ jobs:
           sudo apt-get update
           sudo apt-get install -y s3cmd
 
+      - name: Setup Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: 3.8
+
+      - name: Run Integration tests
+        run: |
+          mkdir -p integration/storage
+          CONTAINER=$(docker run -d \
+            -v $GITHUB_WORKSPACE/integration/storage:/data \
+            -v $GITHUB_WORKSPACE/ceph/build/bin:/radosgw/bin \
+            -v $GITHUB_WORKSPACE/ceph/build/lib:/radosgw/lib \
+            -p 7480:7480 \
+            quay.io/s3gw/run-radosgw \
+              --rgw-backend-store sfs \
+              --debug-rgw 1)
+
+          sleep 5
```

Comment:
Optional:

wrote this a while ago for the same purpose ;)

```
wait_http_200 () {
    local url="$1"

    echo "waiting for $url to return 200:"
    while [[ $(curl -s -o/dev/null -w '%{http_code}' "$url") != "200" ]]; do
      sleep 1
    done
    echo "done"
}
```

Reply:
I'll add one like that, but with the extra feature of having a maximum timeout, so in case the s3gw never comes up we won't block indefinitely.